torch/csrc/jit/runtime/register_ops_utils.cpp

```diff
@@ -428,9 +428,14 @@ void listMulIntRight(Stack* stack) {
 }
 
 void listSlice(Stack* stack) {
-  int64_t step = pop(stack).to<int64_t>();
-  int64_t end = pop(stack).to<int64_t>();
-  int64_t start = pop(stack).to<int64_t>();
+  auto step_val = pop(stack);
+  auto end_val = pop(stack);
+  auto start_val = pop(stack);
+
+  int64_t step = step_val.isInt() ? step_val.to<int64_t>() : 1;
```

Comment:
I would add a comment for why we're setting the values to INT64_MAX if they're None, it's confusing without context.

Reply:
sounds good